tests/components/bthome/test_event.py

```diff
@@ -0,0 +1,116 @@
+"""Test the BTHome events."""
+
+import pytest
+
+from homeassistant.components.bthome.const import DOMAIN
+from homeassistant.components.event import ATTR_EVENT_TYPE
+from homeassistant.const import ATTR_FRIENDLY_NAME, STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+
+from . import make_bthome_v2_adv
+
+from tests.common import MockConfigEntry
+from tests.components.bluetooth import (
+    BluetoothServiceInfoBleak,
+    inject_bluetooth_service_info,
+)
+
+
+@pytest.mark.parametrize(
+    ("mac_address", "advertisement", "bind_key", "result"),
+    [
+        (
+            "A4:C1:38:8D:18:B2",
+            make_bthome_v2_adv(
+                "A4:C1:38:8D:18:B2",
+                b"\x40\x3A\x00\x3A\x01\x3A\x03",
+            ),
+            None,
+            [
+                {
+                    "entity": "event.test_device_18b2_button_2",
+                    ATTR_FRIENDLY_NAME: "Test Device 18B2 Button 2",
+                    ATTR_EVENT_TYPE: "press",
+                },
+                {
+                    "entity": "event.test_device_18b2_button_3",
+                    ATTR_FRIENDLY_NAME: "Test Device 18B2 Button 3",
+                    ATTR_EVENT_TYPE: "triple_press",
+                },
+            ],
+        ),
```

Comment:
Should we be getting 2 or 3 buttons here ?

Reply:
This is the event we get back from `bthome-ble`

```
['update.events',
 {DeviceKey(key='button_3', device_id=None): Event(device_key=DeviceKey(key='button_3',
                                                                        device_id=None),
                                                   name='Button 3',
                                                   event_type='triple_press',
                                                   event_properties=None),
  DeviceKey(key='button_2', device_id=None): Event(device_key=DeviceKey(key='button_2',
                                                                        device_id=None),
                                                   name='Button 2',
                                                   event_type='press',
                                                   event_properties=None)}]
```

Reply:
Confirmed this is ok as button 1 has not event

Reply:
Yep, there is a button 1, but that one is not triggered.